### ISSUE TITLE
Running installers from default download location can fail with access denied

### ISSUE THREAD

**Issue by JohnMcPMS** [microsoft/winget-cli#1650]
### Brief description of your issue

When the user's `%temp%` location does not grant the *medium IL* user token sufficient privileges to execute files, some types of installers will fail to run with an access denied type error (see microsoft/PowerToys#13988).

### Steps to reproduce

For example, set the ACLs on the user's `%temp%` as:
```
>icacls %temp%
%temp% BUILTIN\Administrators:(F)
       BUILTIN\Administrators:(OI)(CI)(IO)(F)
       Everyone:(R,W,DC)
       Everyone:(OI)(CI)(IO)(GR,GW,DC)
```

### Expected behavior

Installers should still run despite the state of the user's `%temp%`.

### Actual behavior

Installers that require execution can fail.

### Environment

```shell
Any winget version since I moved the download location into %temp%.
```

**Comment by Trenly** [comment]
@JohnMcPMS - Was this resolved with #2324 ?

**Comment by JohnMcPMS** [comment]
Yes, it should have been.
